km_api/know_me/factories.py

```diff
@@ -31,6 +31,16 @@ class Meta:
         model = models.EmergencyItem
 
 
+class EmergencyContactFactory(factory.django.DjangoModelFactory):
+    """
+    Factory for generating ``EmergencyContact`` instances.
+    """
+    km_user = factory.SubFactory('know_me.factories.KMUserFactory')
+
```

Comment:
The factory should define defaults for all required fields. I would suggest something like:

```python
class EmergencyContactFactory(factory.django.DjangoModelFactory):
    km_user = factory.SubFactory('know_me.factories.KMUserFactory')
    name = factory.Sequence(lambda n: 'Emergency Contact {n}'.format(n=n))
    relation = 'Caregiver'
    phone_number = '1238675309'
```